akka-typed/src/main/scala/akka/typed/ActorRef.scala

```diff
@@ -74,6 +68,15 @@ abstract class ActorRef[-T](_path: a.ActorPath) extends java.lang.Comparable[Act
 }
 
 object ActorRef {
+
+  implicit final class ActorRefScalaTell[-T](val ref: ActorRef[T]) extends AnyVal {
```

Comment:
if we do this we should name this in more general way, such as `ActorRefOps`, so that more methods can be added if needed